addons/account/tests/test_account_move_line_tax_details.py

```diff
@@ -329,7 +329,7 @@ def test_affect_base_amount_2(self):
                 },
                 {
                     'base_line_id': base_lines[1].id,
-                    'tax_line_id': tax_lines[1].id,
+                    'tax_line_id': tax_lines[0].id,
```

Comment:
Note: this has changed because we compute `tax_ids` on tax lines by traveling the taxes in the reversed sorted order instead of the sorted order as before. It makes the order of taxes in the cache in a different order leading to this change. However, nothing changes at all except tax_line 0 and 1 are not in the same order.